src/pipecat/services/google.py

```diff
@@ -40,7 +40,7 @@
 except ModuleNotFoundError as e:
     logger.error(f"Exception: {e}")
     logger.error(
-        "In order to use Google AI, you need to `pip install pipecat-ai[google]`. Also, set `GOOGLE_API_KEY` environment variable."
+        "In order to use Google AI, you need to `pip install pipecat-ai[google]`. Also, set `GOOGLE_APPLICATION_CREDENTIALS` environment variable."
```

Comment:
The `GOOGLE_API_KEY` is required to run the GoogleLLMService. So, I think this message needs to read: `Also, set `GOOGLE_API_KEY` for the `GoogleLLMService` and `GOOGLE_APPLICATION_CREDENTIALS` for the `GoogleTTSService`.

Reply:
Ah, OK.